File: 프로그래머스/0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.js
function solution(balls, share) {
    let n = 1;
    let m = 1;
    let n_m = 1;
    
    for(let i = 1; i<=balls; i++){
        n *= i;
    }
    for(let i = 1; i<=share; i++){
        m *= i;
    }
    for(let i = 1; i<=(balls-share); i++){
        n_m *= i;
    }
    
    return Math.round(n / (m*n_m));
    
}

File: 프로그래머스/0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.py
def solution(balls, share):
    n = 1
    m = 1
    n_m = 1
    for i in range(1, balls+1):
        n *= i
    for j in range(1, share+1):
        m *= j
    for k in range(1, balls-share+1):
        n_m *= k
    answer = n / (n_m*m)
    return answer